google/cloud/batch/v1/.repo-metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "batch.googleapis.com",
+    "api_shortname": "batch",
+    "client_documentation": "https://cloud.google.com/cpp/reference/batch/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "",
```

Comment:
remove?

Reply:
I was able to find it by browsing the components (the documentation has nothing about support, sigh)